tmt/steps/provision/local.py

```diff
@@ -72,6 +72,7 @@ def execute(self,
                 env: Optional[tmt.utils.EnvironmentType] = None,
                 friendly_command: Optional[str] = None,
                 test_session: bool = False,
+                tty: bool = False,
```

Comment:
I'd emit a warning when `tty` would be set, with a shortened version of the text from specification, something along the lines of `tty requested but not supported by local plugin".

Reply:
ack, will do

Reply:
@happz added warning

```
    execute
        queued execute task #1: default-0 on default-0
        
        execute task #1: default-0 on default-0
        how: tmt
        progress: /test/python [1/2]        warn: ignoring requested tty, not supported by local plugin
        progress: /test/shell [2/2]         warn: ignoring requested tty, not supported by local plugin
        progress:                  
    
        summary: 2 tests executed
```